### PR TITLE
Fix: 'Coding Coach' text is coming in a single line. 

### PR DESCRIPTION
**Pull request by linux-nerd**
# Details

## Description

- [x] Changed 'Coding\nCoach' to 'Coding Coach'
- [x] Added styles from #113 to break the words in multiple lines
- [x] Removed unused assets/home.scss file
- [x] Renamed Home.scss to home.scss and moved it inside assets directory.
- [x] Changed `@media screen and (min-width: 768px)` to `@include respond-to(pc)`
- [x] Added responsive handler for screen size 1024 `@media screen and (min-width: 768px)` - > 
```
 $break-xlarge: 1024px;
------
 @else if $media==largepc {
    @media (min-width: $break-xlarge) {
        @content;
    }
  }
```
- [x] Changed rem to px for padding and margins, considering default pixel size of 16px
## Issue

#128

### CODE REVIEW

File: src/pages/home/assets/home.scss [hunk 1]
```diff
@@ -1,57 +1,148 @@
...
+    }
 
-.mainHeaderBackground {
-  background: url('./main-header-background.png') no-repeat;
-  height: 100%;
-  width: 100%;
-  background-size: cover;
-  position: absolute;
-  z-index: -1; 
+    @media screen and (min-width: 768px) {
```
Comment (crysfel) [inline src/pages/home/assets/home.scss]: Please use the `respond-to` mixing that we already have: https://github.com/Coding-Coach/coding-coach/blob/development/src/theme/base/_mediaqueries.scss#L36

File: src/pages/home/assets/home.scss [hunk 2]
```diff
@@ -1,57 +1,148 @@
...
-.mainHeaderContent {
-  display: flex;
-  margin-left: 4em;
+.hero {
+    position: relative;
+    min-height: 50vh;
+    padding-top: 6rem;
+    padding-bottom: 3rem;
+
+    @media screen and (min-width: 768px) {
```
Comment (crysfel) [inline src/pages/home/assets/home.scss]: For all media queries use `respond-to`

**Comment by emmawedekind** [comment]
I've restarted this build.

**Comment by arthurbuhl** [comment]
Build for this PR will still fail, as the build fix (#136) is not merged yet. After merging that PR, you can rebuild again and then this build will pass (if there are no issues in tests, etc)

**Comment by crysfel** [comment]
Please rebase with latest in development

File: src/pages/home/assets/home.scss [hunk 3]
```diff
@@ -1,57 +1,149 @@
...
+    }
 }
 
-.mainHeaderContent {
-  display: flex;
-  margin-left: 4em;
+.hero {
+    position: relative;
+    min-height: 50vh;
+    padding-top: 6rem;
```
Comment (emmawedekind) [inline src/pages/home/assets/home.scss]: Please use `px` for margin & padding.

Comment (emmawedekind) [reply src/pages/home/assets/home.scss]: This is in multiple instances, but let's be consistent :)

Comment (linux-nerd) [reply src/pages/home/assets/home.scss]: This is a carry forward from #73 :)
I will fix this :)

File: src/theme/base/_mediaqueries.scss [hunk 4]
```diff
@@ -69,6 +71,11 @@ $break-large: 992px;        // tablets
         @content;
     }
   }
+  @else if $media==largepc {
```
Comment (crysfel) [inline src/theme/base/_mediaqueries.scss]: With #123 merged, we no longer need this, can you please update your code accordingly? Thanks!

Comment (tureey) [reply src/theme/base/_mediaqueries.scss]: That's why I said in frontend channel that until we got the new mediaqueries.. everyone should have used manual mediaqueries doing mobile-first. Now is going to be a pain in the ass for some people :disappointed: 

For any help, reach me on slack (victor ribero)

Comment (linux-nerd) [reply src/theme/base/_mediaqueries.scss]: 😞

**Comment by linux-nerd** [comment]
@crysfel I have somehow merged the development branch and fixed the conflicts 😀

**Pull request merged**